src/components/Login/Login.tsx

```diff
@@ -19,13 +22,30 @@ export const Login = () => {
 	const [emailServerError, setEmailServerError] = useState('');
 	const [passwordServerError, setPasswordServerError] = useState('');
 	const navigate = useNavigate();
+	const dispatch = useDispatch<AppDispatch>();
 
 	const {
 		register,
 		handleSubmit,
 		formState: { errors },
 	} = useForm<Inputs>();
 
+	const loginErrors = useSelector(getErrors);
+	const isAuth = useSelector(getIsAuth);
+
+	useEffect(() => {
+		if (isAuth) {

```

Comment:
It is a good idea to move this functionality to a higher component and to load Login or Courses depending on isAuth